src/test/java/gov/usds/case_issues/authorization/CustomAccessDeniedHandlerTest.java

```diff
@@ -0,0 +1,27 @@
+package gov.usds.case_issues.authorization;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.AccessDeniedException;
+
+public class CustomAccessDeniedHandlerTest {
+
+	@Test
+	public void handle_called_ResopnseForbiden() throws IOException, ServletException {
```

Comment:
Ironically, the only thing we care about here is a side-effect that isn't tested. Though I'm not sure there's a good way to test that, given that it's a `static final` logger object—maybe wire up an in-memory appender? Seems potentially annoying.

Reply:
Yeah it is defiantly hard to test right now. It might be easier if we build a json log builder and test that it is called with the correct information. Could also be used to ensure that the format here is the same as the logs generated by logback